### PR TITLE
Add CLU job to BitBucket

### PR DESCRIPTION
**Pull request by ataylorme**
See #283

### CODE REVIEW

File: bitbucket-pipelines.yml [hunk 1]
```diff
@@ -111,3 +111,18 @@ pipelines:
...
+    - step:
+        name: Composer Lock Update
+        services:
+          - docker
+        caches:
+          - docker
+          - composer
+        script:
+          - *bash_env_export
+          - *bash_env_source
```
Comment (aaronbauman) [inline bitbucket-pipelines.yml]: I replaced these two lines with
`          - /build-tools-ci/scripts/set-environment`
 to get the job to build in Pipelines

Comment (ataylorme) [reply bitbucket-pipelines.yml]: @aaronbauman good catch. Since there is no prior job the Bash artifact is not preserved/available to the CLU job

Comment (ataylorme) [reply bitbucket-pipelines.yml]: Updated in `2aa4be2`

**Comment by pantheon-upstream** [comment]
Visual regression test passed! 

View the [visual regression test report](https://circleci.com/gh/pantheon-systems/example-drops-8-composer/3612/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by pantheon-upstream** [comment]
Visual regression test passed! 

View the [visual regression test report](https://circleci.com/gh/pantheon-systems/example-drops-8-composer/3618/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Pull request merged**